projects/media-viewer/src/lib/annotations/comment-set/comment/comment.component.ts

```diff
@@ -95,7 +95,7 @@ export class CommentComponent implements OnChanges {
   }
 
   public onSave() {
-    this._comment.content = this.fullComment;
+    this._comment.content = this.fullComment.length > 5000 ? this.fullComment.slice(0, 5000) : this.fullComment;
```

Comment:
this can just be `this.fullComment.substring(0, 5000);` if the string is less than 5000 characters substring and slice will use the max string length. I'd say use substring as it's more intuitive to what is happening.

Reply:
Also, maybe that value of 5000 should be put into a const somewhere so it doesn't need to be changed twice